docs/en/ingest-management/elastic-agent/configuration/autodiscovery/kubernetes-conditions-autodiscover.asciidoc

```diff
@@ -84,6 +86,13 @@ To set the target host dynamically for a targeted Pod based on its labels, use a
   condition: ${kubernetes.labels.component} == 'kube-scheduler'
 ----
 
+NOTE: Pods labels and annotations can be used in autodiscover conditions. In the case of labels or annotations that include dots(`.`), they can be used in conditions exactly as
+they are defined in the pods. For example `condition: ${kubernetes.labels.app.kubernetes.io/name} == 'ingress-nginx'`. This should not be mistaken with optional dedoted labels and annotations
```

Comment:
Do you think can we enhance the examples here?
We like users to understand what is the actual label and what is the condition.

Also an example with annotation?

I think we should so a condition example before and after 8.6.0

Reply:
To me, there is already an example with the usage of labels in conditions. It is mentioned above that `The condition ${kubernetes.labels.app} == 'redis' will make the Elastic Agent look for a Pod with the label app:redis within the scope defined in its manifest.`
The more complex labels that are not dedoted any more is not a new case. It is just another label to be used. I believe that this note explains it. 
@ChrsMark what do you think ?

Reply:
From my perspective this paragraph achieves the goal to explain how labels should be used in conditions specially regarding the dedoting concpet. Maybe we can one extra example here for an annotation or provide a bunch of examples in a list like:

```
1. To target a Pod labeled with `app: apache` use the condition X
2. To target a Pod with the annotation .... use ....
```
It's up to you to decide how detailed you want to make it.

Reply:
I can add some examples of conditions with complex labels, annotations and host.ip without having to create detailed examples that do not offer a lot.

Reply:
@gizas , @ChrsMark see https://github.com/elastic/observability-docs/pull/2552/commits/23a43822c666455bcd10fd70344abfd2ae6a0e46

Reply:
LGTM now